lib/RequestRouter.js

```diff
@@ -336,7 +337,7 @@ function validateAndCastObject(object, schema, parentKey) {
             for (var arrayKey in value) {
                 arrayValue = validateAndCastParam.call(this, key, value[arrayKey], schemaProperty.items);
                 if (arrayValue && arrayValue.errorCode) {
-                    this.addPropertyError(parentKey + key, arrayValue.errorCode);
+                    this.addPropertyError(parentKey + key + '.' + arrayKey, arrayValue.errorCode);
```

Comment:
@djvirgen Should we return for example the array key on error property: paramArray.0 in the error object or just property: paramArray?